pom.xml

```diff
@@ -225,51 +238,77 @@
         <artifactId>versions-maven-plugin</artifactId>
         <version>${maven.versions.plugin.version}</version>
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-checkstyle-plugin</artifactId>
+      </plugin>
     </plugins>
   </build>
-  <dependencies>
+  <dependencyManagement>
     <!-- Logging -->
-    <dependency>
-      <groupId>org.slf4j</groupId>
-      <artifactId>slf4j-api</artifactId>
-      <version>${slf4j.version}</version>
-    </dependency>
-    <!-- Yammer -->
-    <dependency>
-      <groupId>com.yammer.metrics</groupId>
-      <artifactId>metrics-core</artifactId>
-      <version>${yammer.version}</version>
-    </dependency>
-    <!-- Metrics Client -->
-    <dependency>
-      <groupId>com.arpnetworking.metrics</groupId>
-      <artifactId>metrics-client</artifactId>
-      <version>${metrics.client.version}</version>
-    </dependency>
-    <!-- Test - General -->
-    <dependency>
-      <groupId>org.hamcrest</groupId>
-      <artifactId>java-hamcrest</artifactId>
-      <version>${hamcrest.version}</version>
-      <scope>test</scope>
-    </dependency>
-    <dependency>
-      <groupId>junit</groupId>
-      <artifactId>junit</artifactId>
-      <version>${junit.version}</version>
-      <scope>test</scope>
-    </dependency>
-    <dependency>
-      <groupId>org.mockito</groupId>
-      <artifactId>mockito-core</artifactId>
-      <version>${mockito.version}</version>
-      <scope>test</scope>
-    </dependency>
-  </dependencies>
-  <distributionManagement>
-    <repository>
-      <id>thirdparty</id>
-      <url>http://nexus-dev/content/repositories/thirdparty</url>
-    </repository>
-  </distributionManagement>
+    <dependencies>
+      <dependency>
+        <groupId>org.slf4j</groupId>
+        <artifactId>slf4j-api</artifactId>
+        <version>${slf4j.version}</version>
+      </dependency>
+      <!-- Application -->
+      <dependency>
+          <groupId>io.dropwizard.metrics</groupId>
+          <artifactId>metrics-core</artifactId>
```

Comment:
This depends on both the yammer metrics core and the dw metrics core; is that right?